src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(a) {
+    if (a === undefined) {
+      const resAdder = sum;
+
+      sum = 0;
+
+      return resAdder;
+    } else {
+      sum += a;
+
+      return adder;
+    }
+  };
```

Comment:
You don't need to use "else" operator here if you returned something on 17 lines.